Command/CreateTweetContentCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * File containing the CreateTweetContentType class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace EzSystems\TweetFieldTypeBundle\Command;
```

Comment:
I'd move the command(s), as well as the XML payloads, to a separate commit (text suggestion: `CLI and REST helpers that create the content-type and content`), positioned after Implement the Legacy Storage Engine Converter. I suggest this position as they come usable at this point.

They should somehow come with explanations (maybe in the README) about how you use them (mostly for REST).